app/models/user.rb

```diff
@@ -15,6 +15,7 @@ class User < ApplicationRecord
   has_many :completed_lessons, through: :lesson_completions, source: :lesson
   has_many :projects, dependent: :destroy
   has_many :user_providers, dependent: :destroy
+  belongs_to :track
```

Comment:
@mindovermiles262 This makes an assumption a user is only going to be associated with a single track. Is that a right assumption ? :thinking: